src/main/java/org/openrewrite/java/migrate/lang/var/UseVarForObject.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.lang.var;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.jetbrains.annotations.NotNull;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.config.RecipeExample;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.search.HasJavaVersion;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class UseVarForObject extends Recipe {
+    @Override
+    public String getDisplayName() {
+        return "UseVarForObjects";
+    }
+
+    @Override
+    public String getDescription() {
+        //language=markdown
+        return "Try to apply local variable type inference `var` to variables containing Objects where possible." +
+               "This recipe will not touch variable declaration with genrics or initializer containing ternary operators.";
+    }
+
+    @Override
+    public List<RecipeExample> getExamples() {
+        RecipeExample recipeExample = new RecipeExample();
+        List<RecipeExample.Source> examples = new ArrayList<>();
```

Comment:
I think we typically do this by annotating representative tests with `@DocumentExample`.

Reply:
I suggest we follow that approach and remove this method override; unless there's a convincing reason.

Reply:
Removed the `Recipe#getExamples` overwrite and add `@DocumentExample` to